cli/deduplifhirLib/normalization.py

```diff
@@ -247,6 +249,29 @@ def british_to_american(input_text):
     return input_text
 
 
+def normalize_date_text(input_text):
+    """
+    Normalizes the given date string
+
+    Arguments:
+        input_text: the input date text to normalize
+    
+    Returns:
+        The normalized date string
+    """
+    try:
+        d = date_parser.parse(input_text)
+    except ParserError as e:
+        print(f"Error when trying to parse date {input_text}")
+        print(f"Error: {e}")
+        return ""
+    except OverflowError as e:
+        print("Overflow error when trying to parse date!")
+        print("Input string too long!")
+        print(f"Error: {e}")
+        return ""
```

Comment:
Maybe we can just return the original string on the off-chance that it's close enough that Splink can match on that?

Reply:
That makes sense.